src/frontends/tensorflow/tests/test_models/gen_scripts/empty_tensor_list.py

```diff
@@ -0,0 +1,18 @@
+# Copyright (C) 2018-2023 Intel Corporation
```

Comment:
Just a question: is it some kind of model generator which you suggested to use for Saved Model tests?

Reply:
yes, correct. It generates test model.

Reply:
@gkrivor, absolutely, you can use the same approach for SM format.